rss2email/command.py

```diff
@@ -62,6 +62,9 @@ def add(feeds, args):
     if not feed.to:
         raise _error.NoToEmailAddress(feed=feed, feeds=feeds)
     feeds.save()
+    if args.fetch:
+        feed.run(send=False, clean=False)
+        feeds.save()
 
```

Comment:
There is only a need to call `feeds.save()` once, at the end:
```python
    if not feed.to:
        raise _error.NoToEmailAddress(feed=feed, feeds=feeds)
    if args.fetch:
        feed.run(send=False, clean=False)
    feeds.save()

```
If we save the configuration file only once _after_ fetching the feed, then this also gives an answer to [@auouymous](https://github.com/rss2email/rss2email/pull/182#issuecomment-827204209): without this patch, there is a race after `r2e add`, between the subsequent interactive `r2e run -n` and the background scheduled `r2e run`. So, this patch would save me both some typing _and_ the need to be careful and look at the clock while adding a feed.